examples/flava/train.py

```diff
@@ -25,10 +25,10 @@ def main():
         seed_everything(config.training.seed, workers=True)
 
     datamodules = []
-    imagenet_datamodule = ImageDataModule(
-        **build_datamodule_kwargs(config.datasets.image, config.training)
-    )
     if "image" in config.datasets.selected:
+        imagenet_datamodule = ImageDataModule(
```

Comment:
You need this for `MultimodalEvalCallback` below. That's why it is always initialized.

Reply:
moved it back